idd/Energy+.idd.in

```diff
@@ -43320,11 +43341,67 @@ Coil:Heating:DX:SingleSpeed,
        \maximum 6
        \default 4
        \note Standard Region number for which HSPF and other standard ratings are calculated
-  A13; \field Evaporator Air Inlet Node Name
+  A13, \field Evaporator Air Inlet Node Name
        \type node
        \note Enter the name of an outdoor air node. This node name is also specified in
        \note an OutdoorAir:Node or OutdoorAir:NodeList object.
-
+  A14, \field Zone Name for Evaporator Placement
+       \note This input field is name of a conditioned or unconditioned zone where the secondary
+       \note coil (evaporator) of a heat pump is to be placed.  This is an optional input field 
+       \note specified only when user desires to extract heat from the zone.  The heat extracted 
+       \note is modelled as internal gain of the zone. If the primary DX system is a heat pump, 
+       \note then the zone name should be the same as the zone name specified for placing the 
+       \note secondary cooling DX coil.
+  N13, \field Secondary Coil Air Flow Rate
+       \type real
+       \units m3/s
+       \minimum> 0.0
+       \autosizable
+       \note This input value is the secondary coil (evaporator) air flow rate when the heat pump 
+       \note is working in heating mode or the secondary DX coil (condenser) air flow rate when the 
+       \note heat pump is working in cooling mode.  
+  N14, \field Secondary Coil Fan Flow Scaling Factor
+       \type real
+       \units m3/s
+       \minimum> 0.0
+       \note This input field is scaling factor for autosizing the secondary DX coil fan flow rate.  
+       \note The secondary air flow rate is determined by multiplying the primary DX coil rated air 
+       \note flow rate by the fan flow scaling factor. Default value is 1.25. If the secondary coil 
+       \note fan flow rate is not autosized, then the secondary coil fan flow scaling factor is set 
+       \note to 1.0.
+  N15, \field Nominal Sensible Heat Ratio of Secondary Coil
+       \type real
+       \units m3/s
+       \minimum> 0.0
+       \maximum> 1.0
```

Comment:
@Nigusse \maximum> 1.0 is not valid.  Should be \maximum 1.0
Please fix all five occurrences in the idd.  Will need a new branch/pull request since this was already merged, but I'm not going to bother posting an issue.